packages/components/time-picker/__tests__/time-picker.test.ts

```diff
@@ -727,3 +772,16 @@ describe('TimePicker(range)', () => {
     })
   })
 })
+
+describe('test Format fun', () => {
+  it('extractDateFormat', () => {
+    formatList.forEach((format, i) => {
+      expect(extractDateFormat(format)).toBe(dateArr[i])
+    })
+  })
+  it('extractTimeFormat', () => {
+    formatList.forEach((format, i) => {
+      assert.equal(extractTimeFormat(format), timeArr[i])
+    })
+  })
+})
```

Comment:
I would suggest using `it.each` for cases like this.

Ref: https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
get, I'm sure I should use jtest's syntax, adjust it right away.